subgraphs/erc20/schema.graphql

```diff
@@ -0,0 +1,293 @@
+# Subgraph Schema: ERC20 Tokens
+# Version: 1.0.0
+
+#############################
+####### Token Metadata ######
+#############################
+
+type Token @entity {
+  " Smart contract address of the token "
+  id: ID!
+
+  " Name of the token, mirrored from the smart contract "
+  name: String!
+
+  " Symbol of the token, mirrored from the smart contract "
+  symbol: String!
+
+  " The number of decimal places this token uses, default to 18 "
+  decimals: Int!
+
+  " If token transfers are paused, derived from paused event "
+  isPaused: Boolean
+
+  " Token Holders Balance "
+  holdersBalance: [AccountBalance!]! @derivedFrom(field: "token")
+
+  " Number of token holders "
+  holderCount: BigInt!
+
+  " Total number of token transfer transactions "
+  transferCount: BigInt!
+
+  " Total number of token mint transactions "
+  mintCount: BigInt!
+
+  " Total number of token burn transactions "
+  burnCount: BigInt!
+
+  " Total token supply "

```

Comment:
Should we have a `liquidSupply` or `currentSupply` to track the supply balance? This will be useful in the daily snapshots

Reply:
what is the difference between the `totalSupply` and the `currentSupply`?

Reply:
I'd imagine some of the token that got minted have never been moved (e.g. locked). So would be considered illiquid supply. But I guess those would be difficult to track